miner/src/pool/verifier.rs

```diff
@@ -43,6 +43,8 @@ pub struct Options {
 	pub block_gas_limit: U256,
 	/// Maximal gas limit for a single transaction.
 	pub tx_gas_limit: U256,
+	/// Skip check checks
+	pub tx_queue_no_early_reject: bool,
```

Comment:
Could be named just `no_early_reject`, we are in the context of `tx_queue` already.

Also the comment could be a bit more clear :) Maybe something like:
```
Skip checks for early rejection, to make sure that local transactions are always imported.
```

Reply:
Sorry, that comment was a leftover type. Will fix it & also fix the failing tests.